dist/index.js

```diff
@@ -54,7 +54,7 @@ async function getChangedFilesPaths(token) {
   const parsedDiff = gitDiffParser(pullRequestDiff);
 
   const changedFilePaths = parsedDiff.map((e) => {
-    return e['newPath'].replace('b/', '');
+    return e['newPath'].replace(/^b\//, '');
```

Comment:
Why is this needed in the first place?

Reply:
The git diff parser outputs the file paths in the following format:
1. Files that are modified: `{'oldPath': 'a/path/to/file', 'newPath': 'b/path/to/file'}`
2. Files that are remaned or added`{'oldPath': null, 'newPath': 'b/path/to/file'}`

Hence the need to remove the first occurrence of `b/` from the file path.

Reply:
Actually, the `b/` is supposed to be there for all `newPath` values according to my tests but for some reason it probably was not there for the specific case that you sent me with the bug which is odd.